go/worker/compute/executor/committee/node.go

```diff
@@ -797,6 +851,16 @@ func (n *Node) handleScheduleBatch(force bool) {
 		return
 	}
 
+	// Update per round scheduler parameters.
+	// XXX: could avoid querying here on every round if runtimes would return
+	// next round limits in last execute tx response.
```

Comment:
To make things simpler they could also just return a flag whether the limits have changed and in case they have, we would query the new limits.

Reply:
Would `ExecuteBatchResult` be the right place to add that flag?

Reply:
Yes, that sounds right.

Reply:
Hm actually having a flag might actually not be simpler than returning the limits in `RuntimeExecuteTxBatchResponse` as we'd need to handle:
- worker failed to query limits initially (or never did the query since it wasn't the txnscheduler), but the scheduled transaction batch was limits so the scheduled batch was valid - even if `RuntimeExecuteTxBatchResponse` returned no changes to the limits, the worker needs to re-query these.
- runtimes now need to implement both the query to the limits and also track updates to the limits.

Just returning the limits map in `RuntimeExecuteTxBatchResponse` avoids these both cases.

Reply:
Ok, let's go with always querying the limits for now and then open an issue to add returning the limits map in execute response?

Reply:
Oh, i just pushed support for this :-) Check last commit.